### ISSUE TITLE
List of Reg Email Updates:

### ISSUE THREAD

**Issue by fursuit69** [MidwestFurryFandom/rams#400]
Here is where I will post the status of emails I have touched or requested updates to for Registration:

RAMS: **attendee_payment_received** GitHub: mff_rams_plugin/templates/emails/reg_workflow/attendee_confirmation.html 
     Updates: Two updates to email body (made via comments)
     Note: Is this the same as the **attendee_badge_confirmed** email in the Registration emails list in RAMS? Because they look the same. If they aren't the same email, they certainly could be the same.

RAMS: **group_payment_received** GitHub: mff-rams-plugin/blob/mff-staging/mff_rams_plugin/templates/emails/reg_workflow/group_confirmation.html
     Updates:
    Notes:  I'm not sure how this email actually works at this point. Is it specific to Marketplace situations? I see the Marketplace signature at the bottom. Given the other "group" emails (which are different from art show and dealer's den specific emails), I don't think we need this email active UNLESS marketplace specifically uses it (and other group emails).

RAMS: **pc_group_payment_received** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/reg_workflow/promo_code_group_confirmation.html
     Updates:
     Notes: Not sure how this email applies to system currently or where it is used? But it is automatic/active.

RAMS: **attendee_group_reg_confirmation**
     Updates: See "**attendee_payment_received**" updates above.

RAMS: **group_extra_payment_received** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/reg_workflow/group_donation.txt
      Updates:  This email can be de-activated/de-automated.  We do not currently give extra gives for donations to the event.

RAMS: **group_preassign_badges_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/reg_workflow/group_preassign_reminder.txt
     Updates: This email can be de-activated/de-automated. I don't feel like we currently use it for our reg situations.

RAMS: **group_preassign_badges_reminder_last_chance** GitHub: Same as above, group_preassign_reminder.txt
     Updates: This email can be de-activated/de-automated. I don't feel like we currently use it for our reg situations.

Rams: **regular_badge_confirmation** GitHub: mff-rams-plugin/blob/mff-staging/mff_rams_plugin/templates/emails/placeholders/regular.txt
     Updates:  Made update directly in file. 
     Note: We will need to further update this with the Master Account login steps (versus the direct registration confirmation link) once that is in place.

RAMS: **badge_confirmation_reminder** GitHub: mff-rams-plugin/blob/mff-staging/mff_rams_plugin/templates/emails/placeholders/reminder.txt
     Updates:  Made update directly in file
     Notes: We will need to further update this with the Master Account login steps (versus the direct registration confirmation link) once that is in place.

RAMS:  **badge_confirmation_reminder_last_chance** GitHub: Uses same template as above (reminder.txt)

RAMS: **personalized_badge_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/personalized_badges/reminder.txt
     Updates: None, we don't currently use this email.

RAMS: **under_18_parental_consent_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/reg_workflow/under_18_reminder.txt
     Updates:  Made update directly in file.

RAMS: **qrcode_for_checkin** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/reg_workflow/attendee_qrcode.html
     Updates: Made in file. 

RAMS: **band_checklist_inquiry** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/guests/band_notification.txt
     Updates: Not sure that we currently use this email. Can stay as-is I guess.

RAMS: **band_panel_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/guests/band_panel_reminder.txt
     Updates: Not sure that we currently use this email. Can stay as-is I guess.

RAMS: **band_bio_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/guests/band_bio_reminder.txt
     Updates: Not sure that we currently use this email. Can stay as-is I guess.

RAMS: **band_w9_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/guests/band_w9_reminder.txt
     Updates: Not sure that we currently use this email. Can stay as-is I guess.

RAMS: **band_merch_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/guests/band_merch_reminder.txt
     Updates: We don't currently use this email as far as I know. Can stay as-is.

RAMS: **band_charity_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/guests/band_charity_reminder.txt
     Updates: We don't currently use this email as far as I know. Can stay as-is.

RAMS: **band_stage_plot_reminder** GitHub: rams/blob/add-attendee-accounts/uber/templates/emails/guests/band_stage_plot_reminder.txt
     Updates: We don't currently use this email as far as I know. Can stay as-is.

All of those "band" emails may be good for Pprogramming at some future point.  I can certainly see a use for them in the future, but not for 2021.

**Comment by kitsuta** [comment]
** attendee_payment_received** and ** attendee_badge_confirmed** are indeed the same email template. They use different subject lines based on whether the attendee registered through the normal workflow or claimed a Dealer Assistant badge, respectively.

**group_payment_received** is used in the Dealer application process -- once they've been approved and pay for their table and badges, the group leader will receive this email.

**pc_group_payment_received** is safe to leave on, as it will only send emails to a type of registration that MFF does not use.

All the **band_** emails are part of the GuestGroups plugin, which MFF doesn't use.

I see you noted that you made changes directly in several files, but I don't see those changes anywhere in the repository. Did they get uploaded?

**Comment by fursuit69** [comment]
No idea. I saw a thing that let me edit, so I edited then hit "propose changes". So whatever that does.

**Comment by fursuit69** [comment]
I also see this in relation to my "fork" - do I need to hit the new pull request button?
![Forks](https://user-images.githubusercontent.com/90427815/132974062-60a6845c-1229-44e3-a2c1-aeaca6ee8297.png)

**Comment by kitsuta** [comment]
Ahhhh, I see. It looks like your changes are spread across several branches. You'll want to merge them all into your fork's version of the `mff-staging` branch, and then submit a pull request to merge your fork's `mff-staging` branch with the MidwestFurryFandom version of the `mff-staging` branch. You'll need to do that for both the `mff-rams-plugin` repo and the `rams` repo.

This should hopefully serve as a good overview of how the branching/merging process works: https://guides.github.com/introduction/flow/

Or, if you don't want to learn all of that right now, I can set up the pull requests for you!

**Comment by fursuit69** [comment]
If you could do that last part, the 'set up the pull requests for you' part, that would be great appreciated. <3

**Comment by kitsuta** [comment]
Today I learned that I can't create pull requests in other people's forks!

I filed PRs for the two changes in the mff-rams-plugin. I'll be integrating the changes to the base plugin's emails tomorrow, as some of them need to use the mff-rams-plugin repo instead.
